### DISCUSSION TITLE
Maintenance Release 2.10.12

### DISCUSSION THREAD

**Discussion by tkralphs** [coin-or/Cbc#667] [Announcements]
A maintenance release [2.10.12](https://github.com/coin-or/Cbc/releases/tag/releases%2F2.10.12) was made from the `stable/2.10` branch. With this release, binaries are being built for OSX arm64 architectures. See [here](https://github.com/coin-or/Cbc/compare/releases/2.10.11...releases/2.10.12) for changes.

**Comment by MediciBrown** [comment]
Just to avoid any doubt, hope you don't mind a quick clarification question.
Woud you say this most recent version (2.10.12) is also the most suitable one for someone looking to use a stable CBC vesion right now or are there any earlier ones that would have "less features but more stability"?
